test/checkpointctl.bats

```diff
@@ -14,7 +14,15 @@ function checkpointctl() {
 	echo "$output"
 }
 
+run_only_test() {
+	if [ "$BATS_TEST_NUMBER" -ne "$1" ]; then
+		skip
+	fi
+}
+
 function setup() {
+	# run_only_test 49
```

Comment:
Is this to only run certain tests? I was always using

`-f, --filter <regex>      Only run tests that match the regular expression`

To only run one test.

Reply:
Thanks! I've removed this function.